packages/react-query/src/bin/utils/package.ts

```diff
@@ -1,15 +1,13 @@
-import fs from 'fs'
-import path from 'path'
-import { exit } from 'process'
+import fs from 'node:fs'
+import path from 'node:path'
 
-type LoadModuleResult<T> = { exports: T; isSuccess: true } | { exports: undefined; isSuccess: false }
-
-export function loadModule<T>(name: string): LoadModuleResult<T> {
+// eslint-disable-next-line @typescript-eslint/no-unnecessary-type-parameters
+export function loadModule<T>(name: string): T {
   try {
     // eslint-disable-next-line @typescript-eslint/no-require-imports
-    return { exports: require(name) as T, isSuccess: true }
+    return require(name) as T
   } catch {
-    return { exports: undefined, isSuccess: false }
+    throw new Error(`${name} is not found.`)
   }
 }
```

Comment:
Previously, loadModule did not throw errors directly but instead followed a structure where the CLI (node) would terminate at the point where the function was used. However, this approach made testing difficult, as mocking require in Node.js is not possible. For this reason, we decided to modify the utils function to throw errors directly, as it allows for more reliable testing.

P.S. Once this PR is merged, I plan to implement a global error handler for the CLI and custom error classes in the next version.

ref: https://github.com/tj/commander.js?tab=readme-ov-file#override-exit-and-output-handling